metrotranscom/doorway
# fix: add application announcement banner

# Pull Request Template

## Issue Overview

This PR addresses #531 


## Description

This PR utilizes the maintenance banner to display the applications/disclaimer copy and updates the color based on product's requests.

## How Can This Be Tested/Reviewed?

1. Update the maintenance window env variable to something that includes the time of reviewing. (ie. MAINTENANCE_WINDOW=2024-03-20 16:00 -07:00,2024-03-25 16:55 -07:00)
2. Observe the updated banner in each of the languages
3. Spot check the link text in other languages since I had to do some splicing to get the linking in there

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
